pkg/web/src/app/common/components/StatusCondition.tsx

```diff
@@ -22,22 +22,6 @@ export const StatusCondition: React.FunctionComponent<IStatusConditionProps> = (
 }: IStatusConditionProps) => {
   if (!status) return <StatusIcon status="Loading" label="Validating" />;
 
-  const getStatusType = (severity: string): StatusType => {
```

Comment:
That can be a separate PR
a - it also affects the current page, so it's not just related for using the new sdk
b - it's a change that needs it's own issue + PR for visibility

Reply:
Done via #44